### ISSUE TITLE
Remove etherpad contribute

### ISSUE THREAD

**Issue by alperbayata** [ether/etherpad-lite#4374]
How to disable contributing

**Comment by JohnMcLear** [comment]
Wha?

**Comment by alperbayata** [comment]
[url=https://hizliresim.com/TrvNe2][img]https://i.hizliresim.com/TrvNe2.png[/img][/url]  Is it possible to remove this

**Comment by JohnMcLear** [comment]
Yes.  Either with a plugin or custom CSS rule to just hide the button.  The plugin might want to override the /import & /export endpoints too if you want to be thorough about the job.

It can probably also be done in settings.json by modifying the toolbar options.

Please do a tiny bit of effort before creating noise here, we are all very busy and need our community to do some self serving before reaching out to devs.  :heart:

**Comment by alperbayata** [comment]
No. This is not what I mean. How can I remove the "contributor" in the import section every time I open Etherpad. I want to import files directly without contributing.

**Comment by JohnMcLear** [comment]
https://github.com/ether/etherpad-lite/blob/develop/settings.json.template#L483

TLDR; set

``allowAnyoneToImport": true``

**This is not recommended if your instance is not behind some authentication/authorization** -- Setting this to true can allow a malicious actor to spam import endpoints unrestricted.

**Comment by alperbayata** [comment]
Thank you very much. I apologize for my bad english :)

**Comment by JohnMcLear** [comment]
No problem, I went through and corrected where applicable.  You might prefer Discord for basic questions like this.    https://discord.gg/FDhurk
